### ISSUE TITLE
Admin edit template worlds

### ISSUE THREAD

**Issue by denis-ev** [Butzlabben/worldsystem#44]
**Is your feature request related to a problem? Please describe.**
I tried to use the world waka islands 2 and when using it as a template the player gets teleported and instantly dies, when i try to go back after dying my spawn is in nowhere and i have to delete the world

**Describe the solution you'd like**
Pre set custom spawn per custom world

/ws admin get {custom world} -> saves directly to plugin/WorldSystem/worldresources
/ws admin setspawn  -> only works when in the admin world (could be only showing what to paste into the config.yml, needs custom per world spawn tho)
/ws admin home -> always as creative and fly so you can't fall or die

**Comment by Butzlabben** [comment]
You can set the worldspawn in the config.yml.

**Comment by denis-ev** [comment]
Yes I can set **a** spawn, but I need different spawns for different worlds.

This is what your config says
`# Location where you spawn when you join a world`

**Comment by Butzlabben** [comment]
A feature like different fixed spawns for different templates will be implemented
